sdk/storage/azblob/blockblob/models.go

```diff
@@ -23,6 +23,14 @@ type Block = generated.Block
 // BlockList - can be uncommitted or committed blocks (committed/uncommitted)
 type BlockList = generated.BlockList
 
+// BlobCopySourceTags - can be 'COPY' or 'REPLACE'
+type BlobCopySourceTags = generated.BlobCopySourceTags
+
+const (
+	BlobCopySourceTagsCopy    BlobCopySourceTags = "COPY"
+	BlobCopySourceTagsReplace BlobCopySourceTags = "REPLACE"
+)
```

Comment:
This constant type should be added in blockblob/constants.go.
 Also, the constants should be using the value from the generated code like,
```
BlobCopySourceTagsCopy    BlobCopySourceTags = generated.BlobCopySourceTagsCOPY
```

Reply:
Done!